pages.fr/common/git-reset.md

```diff
@@ -0,0 +1,36 @@
+# git reset
+
+> Enlève des commits ou des changements en réinitialisant la tête git à l'état spécifié
+> Si un chemin est passé, git reset fonctionne comme «unstage»;
+> Si un hash de commit est passé, git reset annule les commits jusqu'à ce dernier.
+> Plus d'informations: <https://git-scm.com/docs/git-reset>.
+
+- Tout enlever de la *zone de stage*:
```

Comment:
I remember that in French the grammar use a <space> before the `:`, you can use that rule here:
```suggestion
- Tout enlever de la *zone de stage* :
```